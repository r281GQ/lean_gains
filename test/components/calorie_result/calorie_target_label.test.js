import React from 'react';
import { shallow } from 'enzyme';
import CalorieTargetLabel from './../../../src/components/calorie_result/calore_target_label';

describe('CalorieTargetLabel test', () => {
  it('should render', () => {
    const component = shallow(<CalorieTargetLabel />);
    expect(component).toHaveLength(1);
    expect(component).toMatchSnapshot();
  });
});
