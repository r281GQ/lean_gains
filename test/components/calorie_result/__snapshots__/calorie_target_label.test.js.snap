// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalorieTargetLabel test should render 1`] = `
<FlatButton
  disabled={true}
  fullWidth={false}
  label="0"
  labelPosition="after"
  labelStyle={Object {}}
  onKeyboardFocus={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  onTouchStart={[Function]}
  primary={false}
  secondary={false}
/>
`;

exports[`calorie_target_result label should be 0 when no label provided 1`] = `
<FlatButton
  disabled={true}
  fullWidth={false}
  label="0"
  labelPosition="after"
  labelStyle={Object {}}
  onKeyboardFocus={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  onTouchStart={[Function]}
  primary={false}
  secondary={false}
/>
`;

exports[`calorie_target_result should render label 1`] = `
<FlatButton
  disabled={true}
  fullWidth={false}
  label="label"
  labelPosition="after"
  labelStyle={Object {}}
  onKeyboardFocus={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  onTouchStart={[Function]}
  primary={false}
  secondary={false}
/>
`;
